python/tvm/driver/tvmc/compiler.py

```diff
@@ -95,6 +95,12 @@ def add_compile_parser(subparsers):
         type=common.parse_shape_string,
         default=None,
     )
+    parser.add_argument(
+        "--disabled-pass",
+        help="disable specific passes, comma-separated list of pass names",
+        type=common.parse_disabled_pass,
+        default=None,
```

Comment:
As I mentioned in another comment, you could assign `""` to default, so that the utility function can take only string but not None.